nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSCachingRepository.java

```diff
@@ -203,14 +203,11 @@ public State readState(String id) {
     @Override
     public void createState(State state) {
         repository.createState(state);
-        // TODO check if it's relevant
-        putInCache(state);
```

Comment:
Please add a comment stating that we don't put the state in cache on purpose to avoid polluting it on import.